test/org/test-org.js

```diff
@@ -20,4 +23,239 @@ describe('axway org', () => {
+							"org_id": 100,
+							"data": {
+								"team": {
+									"guid": "60001",
+									"name": "B Team",
+									"default": false
+								},
+								"org_name": "Foo org"
+							}
+						},
+						{
+							"ts": 1613253008000,
+							"message": "Session created for user __s__foo@bar.com__/s__",
+							"event": "platform.session.create",
+							"org_id": 100,
+							"data": {}
+						}
+					]
+				});
+		});
+
+		it('should error if dates are invalid', async function () {
+			initHomeDir('home-local');
+			this.servers = await startServers();
+			await runAxwaySync([ 'auth', 'login' ], { env: { DISPLAY: 1 } });
+
+			let { status, stdout, stderr } = await runAxwaySync([ 'org', 'activity', '--from', 'foo' ]);
+			expect(status).to.equal(1);
+			expect(stderr.toString()).to.match(renderRegexFromFile('activity/bad-from-date'));
+
+			({ status, stderr } = await runAxwaySync([ 'org', 'activity', '--to', 'bar' ]));
+			expect(status).to.equal(1);
+			expect(stderr.toString()).to.match(renderRegexFromFile('activity/bad-to-date'));
+		});
+
+		it('should output activity help', async () => {
+			const { status, stdout } = await runAxwaySync([ 'org', 'activity', '--help' ]);
+			expect(status).to.equal(2);
+			expect(stdout).to.match(renderRegexFromFile('activity/help'));
+		});
+	});
+
+	describe('list', () => {
+		afterEach(stopServers);
+		afterEach(resetHomeDir);
+
+		it('should error if not logged in', async function() {
+			initHomeDir('home-local');
+
+			const { status, stderr } = await runAxwaySync([ 'org', 'list' ]);
+			expect(status).to.equal(1);
+			expect(stderr).to.match(renderRegexFromFile('list/not-authenticated'));
+		});
+
+		it('should list all orgs', async function() {
+			initHomeDir('home-local');
+			this.servers = await startServers();
+			await runAxwaySync([ 'auth', 'login' ], { env: { DISPLAY: 1 } });
+
+			const { status, stdout } = await runAxwaySync([ 'org', 'list' ]);
+			expect(status).to.equal(0);
+			expect(stdout).to.match(renderRegexFromFile('list/foo-bar'));
+		});
+
+		it('should list all orgs as JSON', async function() {
+			initHomeDir('home-local');
+			this.servers = await startServers();
+			await runAxwaySync([ 'auth', 'login' ], { env: { DISPLAY: 1 } });
+
+			const { status, stdout } = await runAxwaySync([ 'org', 'list', '--json' ]);
+			expect(status).to.equal(0);
+			const result = JSON.parse(stdout);
+			expect(result).to.deep.equal({
+				"account": "test_client:foo@bar.com",
+				"orgs": [
+					{
+						"default": false,
+						"guid": "2000",
+						"id": 200,
+						"name": "Bar org"
+					},
+					{
+						"default": true,
+						"guid": "1000",
+						"id": 100,
+						"name": "Foo org"
+					}
+				]
+			});
+		});
+
+		it('should output list help', async () => {
+			const { status, stdout } = await runAxwaySync([ 'org', 'list', '--help' ]);
+			expect(status).to.equal(2);
+			expect(stdout).to.match(renderRegexFromFile('list/help'));
+		});
+	});
+
+	describe('rename', () => {
```

Comment:
are this block + the ones below just placeholders for future tests in pending stories?

Reply:
They are just placeholders. We should have implemented them back when I/we wrote the original functionality, but we never got around to it. So we have a catch all ticket APIGOV-19221 to implement integration tests for everything. Since this ticket touches OUM teams that had no integration tests, I started added a few of the integration tests before I ran out of time. Now we wait until I have time down the road to add more tests.